inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -300,20 +300,20 @@ void GNAGraphCompiler::ConvolutionPrimitive(InferenceEngine::CNNLayerPtr layer)
 
     // TODO: refine following condition
     if (((in_channels > 1) && (in_height > 1) && (in_width > 1)) || // 3D input
-        (convolution._kernel_x != 1 && convolution._kernel_y != 1 && convolution._kernel_y != in_channels) || // 2D kernel
-        (inputs->getLayout() != Layout::NHWC && in_height != 1)) {
+        (convolution._kernel_x != 1 && convolution._kernel_y != 1) || // 2D kernel
+        in_height != 1) {
         // TensorFlow default layout is NHWC
         // OpenVino Default layout is   NCHW
         // GNA Convolution input is     NHCW
         // When layer layout is in NHWC it means that is was created by PassManager
 #if GNA_LIB_VER == 2
         return finalizeConvolution2DPrimitive(layer, in_batch, in_channels, in_height, in_width,
-                                                out_batch, out_channels, out_height, out_width);
+                                              out_batch, out_channels, out_height, out_width);
 #endif
         THROW_GNA_LAYER_EXCEPTION(layer) << "Convolution 2D is not supported on GNA 1.0 library";
     }
     finalizeConvolution1DPrimitive(layer, in_batch, in_channels, in_height, in_width,
```

Comment:
There is no sense to pass `in_height` as always `==1`.

Reply:
height arguments are removed